shopinvader_image/hooks.py

```diff
@@ -40,3 +39,22 @@ def post_init_hook(cr, registry):
         # load data requiring components
         for demo in DEMO_POST_INIT:
             load_xml(env, "shopinvader_image", demo)
+        # prevent removal of demo data since the xml files are not declared
```

Comment:
would be nice to understand why this hook is needed at all... :thinking: 
What about adding a comment on top?

Reply:
@simahawk done